src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,26 +1,35 @@
+<div class="about-modal-pf">
 <div class="modal-header">
   <button type="button"
           class="close pull-right"
           aria-label="Close"
           (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
+    <span class="pficon pficon-close"></span>
   </button>
 </div>
 <div class="modal-body">
-  <h1>ceph</h1>
-  <div class="product-versions">
-    <strong>Ceph version</strong>
-    {{ versionNumber }}
+  <h1>Red Hat Ceph Storage Dashboard</h1>
+  <div class="product-versions-pf">
+    <ul class="list-unstyled">
+      <li><strong>Version</strong> 3.2</li>
```

Comment:
May we use [this solution](https://www.jpkeisala.com/setting-global-constants-angular-2-enviroment/) or [this other one](https://stackoverflow.com/a/47653328) to use global constant strings and group them into a single file (constants.ts)?

```ts
// constants.ts
PRODUCT_NAME = "Red Hat Ceph Storage Dashboard";
PRODUCT_VERSION = "3.2"
PRODUCT_TRADEMARK = `
Licensed under Creative Commons Attribution Share
Alike 3.0 (CC-BY-SA-3.0).
`
```
```html
...
  <h1>{{ PRODUCT_NAME }}</h1>
  <div class="product-versions-pf">
    <ul class="list-unstyled">
      <li><strong>Version</strong> {{ PRODUCT_VERSION }}</li>
...
   <div class="trademark-pf">
     {{ PRODUCT_TRADEMARK }}
   </div>
 ```

Reply:
ack